tests/iostability/test_iostability_degraded_path.py

```diff
@@ -116,3 +124,143 @@ def test_object_crud_single_pod_failure(self):
         self.test_completed = True
         self.log.info("ENDED: Test for Object CRUD operations in degraded mode in loop using "
                       "S3bench for 7 days")
+
+    @pytest.mark.lc
+    @pytest.mark.io_stability
+    @pytest.mark.tags("TEST-40174")
+    def test_degraded_iteration_write_read_partial_delete(self):
+        """Perform 40% Writes of user data capacity (Healthy mode) and perform Object CRUD
+         (40% write,Read, 20% delete) operation(degraded mode)."""
+        self.log.info("STARTED: Test for Perform 40% Writes of user data capacity (Healthy mode) "
+                      "and perform Object CRUD(40% write,Read,20% delete) operation(degraded mode)")
+
+        duration_in_days = self.test_cfg['degraded_path_durations_days']
+        clients = (len(self.worker_node_list)-1) * self.test_cfg['sessions_per_node_vm']
+        write_percent_per_iter = self.test_cfg['test_40174']['write_percent_per_iter']
+        delete_percent_per_iter = self.test_cfg['test_40174']['delete_percent_per_iter']
+        max_cluster_capacity_percent = self.test_cfg['test_40174']['max_cluster_capacity_percent']
+        s3userinfo = dict()
+        s3userinfo['accesskey'] = ACCESS_KEY
+        s3userinfo['secretkey'] = SECRET_KEY
+        bucket_prefix = "test-40174-bkt"
+        end_time = datetime.now() + timedelta(days=duration_in_days)
+
+        self.log.info("Step 1: Get the used percentage of disk space.")
+        total_cap, avail_cap, used_cap = self.health_obj.get_sys_capacity()
```

Comment:
Does `get_sys_capacity()` give values by taking SNS configuration in account?

Reply:
No, get_sys_capacity does not take SNS configuration in account.